.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
The other issue (which IMHO kills the use of `defaultWithAlign`) is that `scalafmt` appears to allow lines to go over the max. line length when auto-indenting. If `scalafmt` was performing better maintenance of the pretty printing auto-indent, I'd find it hard to argue against its use (since all the resulting maintenance burdon would be managed by the plugin code).